main.py

```diff
@@ -2,6 +2,50 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    create_genres()
+    create_actors()
+    update()
+    delete()
+    return get_specific_actors()
+
+
+def create_genres() -> None:
+    for name in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=name)
+
+
+def create_actors() -> None:
+    actors = [
+        "George Klooney",
```

Comment:
You could make it as a list of tuples and so you wouldn't need to use split() method later